src/Engine/StreamStorm/core/SeparateInstance.py

```
@@ -11,19 +10,24 @@
 logger: Logger = getLogger(f"streamstorm.{__name__}")
 
 class SeparateInstance(Playwright):
-    __slots__: tuple[str, ...] = ('channel_name', 'index')
+    __slots__: tuple[str, ...] = ('channel_name', 'index', 'wait_time', 'should_wait')
```

Ruff (0.14.10)

`SeparateInstance.__slots__` is not sorted

Apply a natural sort to `SeparateInstance.__slots__`

(RUF023)